middleware.ml

```diff
@@ -123,21 +128,29 @@ let cookie_value cookie =
   | _ -> Error (`Msg "Bad cookie")
 
 let user_from_auth_cookie cookie users =
-  match cookie_value cookie with
-  | Ok cookie_value -> (
-      match User_model.find_user_by_key cookie_value users with
-      | Some user -> Ok user
-      | None -> Error (`Msg "User not found"))
-  | Error (`Msg s) ->
-      Logs.err (fun m -> m "Error: %s" s);
-      Error (`Msg s)
+  match User_model.find_user_by_key cookie users with
+  | Some user -> Ok user
+  | None -> Error (`Msg "User not found")
 
-let user_of_cookie users now reqd =
+let get_cookie_from_request reqd =
```

Comment:
```suggestion
let auth_cookie reqd =
```

I'd expect when a function gets a `request` that it uses that value for its operations -- no need for `from_request`.

Reply:
done in 69aef56